code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -10,7 +10,7 @@
 	icon_state = "meat"
 	dried_type = /obj/item/reagent_containers/food/snacks/sosjerky/healthy
 	bitesize = 3
-	list_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/cooking_oil = 2) //Meat has fats that a food processor can process into cooking oil
+	list_reagents = list(list(/datum/reagent/consumable/nutriment/protein = 6, /datum/reagent/consumable/cooking_oil = 2)) //Meat has fats that a food processor can process into cooking oil
```

Comment:
```suggestion
	list_reagents = list(/datum/reagent/consumable/nutriment/protein = 6, /datum/reagent/consumable/cooking_oil = 2) //Meat has fats that a food processor can process into cooking oil
```
I assume this was a typo, right?

Reply:
yes i had to manually add these back 1 by 1 so its a miss-copy